src/vcpkg/commands.integrate.cpp

```diff
@@ -620,7 +620,7 @@ namespace vcpkg::Commands::Integrate
         static const std::string FISH = "x-fish";
     }
 
-    static std::vector<std::string> valid_arguments(const VcpkgPaths&)
+    static std::vector<std::string> valid_arguments(const VcpkgPaths& paths VCPKG_UNUSED)
```

Comment:
I think the previous form that worked with compilers before `[[maybe_unused]]` was a thing was better. Ditto elsewhere.